### DISCUSSION TITLE
bind-Value support and/or hiding controls

### DISCUSSION THREAD

**Discussion by Chris-Mingay** [jsakamoto/BlazingStory#18] [Q&A]
Hi there, thanks for your efforts on this I really appreciate it, it's going to be very helpful!

Is there a "proper" way for handing components that make use of e.g. `bind-Value`? Secondly is it possible to hide controls? This ties into those that are usually automatically bound by the use of bind-Value.

```cs
@using WSComponents
@attribute [Stories("Components/WSTextInput")]

<Stories TComponent="WSTextInput">
    <Story Name="Default">
        <Arguments>
            <Arg For="_ => _!.Label" Value='"Super Label"'/>
        </Arguments>
        <Template>
            <WSTextInput @attributes="context.Args" @bind-Value="@TextValue"/>
        </Template>
    </Story>
</Stories>

@code {
    private string TextValue { get; set; } = "Text Value";
}
```

This currently generates the following control set:

![image](https://github.com/jsakamoto/BlazingStory/assets/71371350/adb8ac38-c279-4ff8-b2a1-a96da7c9d550)

If possible I'd like to hide AdditionalAttributes, Value, ValueChanged, ValueExpression and DisplayName.

Thank you
Chris

**Comment by kristoflagae** [comment]
Hi,
I’ve started working with this library and encountered the same questions as mentioned above.
Do you already have more information on this?
Or is using the @bind-value an anti-pattern, because stories are meant to be demonstrative?

In any case, hide some parameter would definitely be interesting
